MAUI/DataGrid/Column-sizing.md

```diff
@@ -0,0 +1,186 @@
+
+<table>
+<tr>
+<th>
+Type
+</th>
+<th>
+Column width
+</th>
+</tr>
+<tr>
+<td>
+<code>Fill</code>
+</td>
+<td>
+Divides the total width equally for columns. 
+</td>
+</tr>
+<tr>
+<td>
+<code>Auto</code>
+</td>
+<td>
+Calculates the width of column based on header and cell contents. So that header and cell content’s are not truncated.  
+</td>
+</tr>
+<tr>
+<td>
+<code>LastColumnFill</code>
+</td>
+<td>
+The column width of the DataGridColumns are adjusted with respect to [SfDataGrid.DefaultColumnWidth](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_DefaultColumnWidth) property. In case the columns does not fill the entire view space, width of the last column fills the unoccupied space in the view.
+</td>
+</tr>
+<tr>
+<td>
+<code>FitByCell</code>
+</td>
+<td>
+Calculates the width of column based on cell contents. So that cell content’s are not truncated.  
+</td>
+</tr>
+<tr>
+<td>
+<code>FitByHeader</code>
+</td>
+<td>
+Calculates the width of column based on header content. So that header content is not truncated.  
+</td>
+</tr>
+<tr>
+<td>
+<code>None</code>
+</td>
+<td>
+Default column width or defined width set to column.
+</td>
+</tr>
+</table>
+
+N> ColumnWidthMode will not work when the column width defined explicitly. ColumnWidthMode calculates column width based on [MinimumWidth](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html#Syncfusion_Maui_DataGrid_DataGridColumn_MinimumWidth) and [MaximumWidth](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html#Syncfusion_Maui_DataGrid_DataGridColumn_MaximumWidth) properties.
+
+Below code, applies [ColumnWidthMode.Fill](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_DefaultColumnWidth) to equally set width for `SfDataGrid.Columns`.
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid  x:Name="dataGrid"
+                        ColumnWidthMode="Fill"
+                        ItemsSource="{Binding Orders}" />
+{% endhighlight %}
+{% endtabs %}
+
+![Fill column width mode in .NET MAUI DataGrid](Images\column-sizing\maui-datagrid-fill-column-width-mode.png)
+
+
+## Apply ColumnWidthMode for a particular column
+
+To apply column sizing to individual column, use the `DataGridColumn.ColumnWidthMode` property. The `DataGridColumn.ColumnWidthMode` property is also a type of the `ColumnWidthMode`. If the `DataGridColumn.ColumnWidthMode` is not explicitly set to a value, then it takes the value of the `SfDataGrid.ColumnWidthMode`and applies width to the columns accordingly.
+
+To apply `ColumnWidthMode` for a particular column, follow the code example:
+
+{% tabs %}
+{% highlight xaml %}
+<sfgrid:SfDataGrid x:Name="dataGrid"
+                   AutoGenerateColumns="True"
```

Comment:
@KarthikRajaKalaimani  - This property (AutoGenerateColumns) is not available. Be aware while you giving the code snippet.

Reply:
Sorry for the inconvenience caused, now removed that property